internal/sabnzbd/model/model.go

```diff
@@ -166,6 +166,10 @@ func (q *QueueStats) UnmarshalJSON(data []byte) error {
 
 // latestStat gets the most recent date's value from a map of dates to values
 func latestStat(m map[string]int) (string, int) {
+	if len(m) == 0 {
```

Comment:
What situation was leading to the map being empty?

I'm a little worried this may actually cause a bug that leads to incorrect metrics. In the collector itself, we compare the cached day string to the one returned from the new Stats object, and if they're not the same, we assume the day has rolled over, and reset the counter:
https://github.com/onedr0p/exportarr/blob/e732d200a9a7cac6f84e9e1f8f1faee74dcb5d58/internal/sabnzbd/collector/cache.go#L25-L40

So if we receive an empty map due to, for example, a sqlite error on the sab side, we could assume the day has changed too early, resulting in double counting of server stats.

I think we can verify behavior here with a quick unit test that:
1. Creates a new statCache
2. calls `Update` with a non-zero `serverStats` object
3. calls `Update` with a zeroed `serverStats` object
4. calls `Update` one more time with the object from step 2.

And then verify that the count is correct.